dateutil/test/test_rrule.py

```diff
@@ -4439,6 +4439,26 @@ def testToStrLongIntegers(self):
                                   byweekno=long(2),
                                   dtstart=datetime(1997, 9, 2, 9, 0)))
 
+    def testToStrTimezone(self):
+        ZERO = timedelta(0)
+
+        class UTC(tzinfo):
```

Comment:
There's already a UTC class, `dateutil.tz.tzutc`, which also hhas a singleton, `dateutil.tz.UTC` available for use.